src/g3log/logcapture.hpp

```diff
@@ -55,12 +55,11 @@ struct LogCapture {
 #	endif
 #else
 #	define G3LOG_FORMAT_STRING
-#endif
 
    // Use "-Wall" to generate warnings in case of illegal printf format.
    //      Ref:  http://www.unixwiz.net/techtips/gnu-c-attributes.html
    [[gnu::format(printf, 2, 3)]] void capturef(G3LOG_FORMAT_STRING const char *printf_like_message, ...); // 2,3 ref:  http://www.codemaestro.com/reviews/18
-
+#endif
```

Comment:
This breaks the usage for windows. The CI shows this by build failure

Reply:
Ops my bad.